libraries/adaptive-expressions/etc/adaptive-expressions.api.md

```diff
@@ -206,7 +206,7 @@ export class BoolExpression extends ExpressionProperty<boolean> {
 
 // @public
 export class BoolExpressionConverter {
-    // Warning: (ae-forgotten-export) The symbol "Input" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "Input_2" needs to be exported by the entry point index.d.ts
```

Comment:
I guess these types of changes are related to the api-compat library upgrade, right?

Reply:
yes